docs/Custom-Brimcap-Config.md

```diff
@@ -160,22 +159,28 @@ analyzers:
   - cmd: /usr/local/bin/suricata-wrapper.sh
 ```
 
-A Brimcap analyzer has the following characteristics:
+To be invoked successfully by Brimcap, an analyzer process should have the
+following characteristics:
+
+1. It expects pcap input to be streamed to it via standard input (stdin).
+2. It is assumed that if stdin is closed on a process launched by the analyzer,
+that process will exit.
+3. The analyzer process must ensure that all processes that have inherited
+stdout/stderr have been killed before the analyzer process exits.
```

Comment:
@mattnibs: Once again, your original wording was:

> The process must ensure that all child processes that have inherited stdout/stderr have been killed before exiting.

I assume my slight rewording is still accurate, but, please confirm. Thanks!